code/datums/gamemode/dynamic/dynamic_rulesets_latejoin.dm

```diff
@@ -197,6 +197,67 @@
 	newpd.AnnounceObjectives()
 	return 1
 
+//////////////////////////////////////////////
+//                                          //
+//                   GRUE                   ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+//                                          //
+//////////////////////////////////////////////
+
+/datum/dynamic_ruleset/latejoin/grue
+	name = "Grue Infestation"
+	role_category = /datum/role/grue
+	enemy_jobs = list()
+	required_enemies = list()
+	required_candidates = 1
+	weight = BASE_RULESET_WEIGHT
+	cost = 25
+	requirements = list(5,5,15,15,20,20,20,20,40,70)
+	high_population_requirement = 10
+	logo = "grue-logo"
+
+	repeatable = TRUE
+	var/grue_spawn_spots=list()
+	var/turf/T
```

Comment:
I am not sure about single-letter vars in datum objects. Within procs it's almost canon, but here it's less excusable.